### DISCUSSION TITLE
Want to Contribute

### DISCUSSION THREAD

**Discussion by UjjwalSainiCDC** [UjjwalSaini07/VedicSadhana#68] [Q&A] [answered]
I want to contribute to this project so how i start

**Comment by UjjwalSaini07** [comment, marked as answer]
Thanx a lot for showing ur Interest in this project!!!
Yes this repo is totally open for open source contributon.

**_Just Read this two Document:_**
1. https://github.com/UjjwalSaini07/VedicSadhana/blob/main/README.md
2. https://github.com/UjjwalSaini07/VedicSadhana/blob/main/CODE_OF_CONDUCT.md

Do good prs and mind it not use main to main merge make new branch to create and create a pr, eg: branch1 to main pr

Please use this approach to contribute in Readme file everything is written.
Thanx

**Comment by UjjwalSainiCDC** [comment]
Thank u i will read all that stuff then i contribute to this
